.github/CODEOWNERS

```diff
@@ -1,2 +1,3 @@
+
```

Comment:
I think this was done by mistake?

Reply:
The template emits it like that. I left it in because I thought having a zero diff is nice when the file is the same as what the template gives.

Reply:
Oh, weird...